beets/autotag/__init__.py

```diff
@@ -144,6 +189,7 @@ def apply_item_metadata(item: Item, track_info: TrackInfo):
 def apply_album_metadata(album_info: AlbumInfo, album: Album):
     """Set the album's metadata to match the AlbumInfo object."""
     _apply_metadata(album_info, album)
+    correct_list_fields(album)
```

Comment:
So this was required to fix the issue with Discogs tagged files that have `albumtypes` missing, right?

But nevertheless, I'll look into adding an `albumtypes` list field to the Discogs plugin. It just makes more sense to store those multiple values we usually get from Discogs accordingly. Haven't looked into your latest plugin changes though! Will do!